test/unix/main.ml

```diff
@@ -776,6 +776,30 @@ module Throttle = struct
     ]
 end
 
+(* Tests of {Data.encode} and {Data.encode'} *)
+module Encode = struct
+  module EDS = I.Private.Data.Entry.Make (KeyDS) (Value)
+  module E = I.Private.Data.Entry.Make (Key) (Value)
+
+  let encode () =
+    let e = E.v "abcdefghijklmnopqrst" "AbcdefghijklmnopqrsT" in
+    let buf = Buffer.create 30 in
+    E.encode e (Buffer.add_string buf);
+    Format.eprintf "%s@." (Buffer.contents buf)
```

Comment:
You could test that the decoding gives you the same value back here.
I understand this is supposed to test that `Entry.encode` doesn't raise an exception here, but the test in itself does not look very meaningful without context (especially in a few months when we forget we had these exceptions).